chapters/inheritance.tex

```diff
@@ -564,6 +564,76 @@ \subsection{Final Element Modification Prevention}\label{final-element-modificat
 \end{lstlisting}
 \end{example}
 
+\subsection{Removing Modifiers -- break}\label{removing-modifiers-break}
+Modifications may contain the special keyword \lstinline!break! instead of an expression.
+The intention of \lstinline!break! is to remove the value.
+
+The modifiers using \lstinline!break! are merged using the same rule as other modifications, and follow the same restrictions so they cannot override a final modifier.
+During flattening of an instantiated model, remaining \lstinline!break! modifications (i.e., the ones that are not further overriden) are treated as if the expression was missing.
+The \lstinline!break! modifier for a variable of a simple type can be applied to the value and/or to specific attributes.
+It can override nothing, either because there was no expression originally or because \lstinline!break! override another \lstinline!break!.
```

Comment:
This sentence is not very clear. "Can override nothing" usually means the same as "cannot override anything". I think this is not the intended meaning, is it?

Reply:
Correct. Will update